src/main/java/BeeBot.java

```diff
@@ -0,0 +1,151 @@
+import parser.Parser;
+import storage.Storage;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import tasklist.TaskList;
+import task.Task;
+import exceptions.EmptyDescriptionException;
+import exceptions.MissingDeadlineException;
+import exceptions.TaskNotFoundException;
+import exceptions.MissingEventTimeException;
+
+
+/**
+ * Represents the main application for managing tasks.
+ * <p>
+ * This class handles user interactions, processes commands, and manages tasks in a list. It also loads and saves tasks
+ * from/to a specified file.
+ * </p>
+ */
+public class BeeBot {
+    private static Storage storage;
+    private static ArrayList<Task> taskList;
+    private static String FILEPATH;
+
+    /**
+     * Constructs a {@code BeeBot} instance with the specified file path.
+     * <p>
+     * Initializes the storage and attempts to load the task list from the provided file path. If loading fails,
+     * initializes an empty task list.
+     * </p>
+     *
+     * @param filePath the path to the file used for loading and saving tasks
+     */
+    public BeeBot(String filePath) {
+        FILEPATH = filePath;
+        storage = new Storage();
+        try {
+            taskList = storage.loadTaskListFromFile(filePath);
+            System.out.println(taskList);
+        } catch (Exception e) {
+            System.out.println("Error loading file: " + e.getMessage());
+            taskList = new ArrayList<>();
+        }
+    }
+
+    /**
+     * Generates a response for the user's chat message.
+     */
+    public static String getResponse(String input) {
+        String[] parts = input.split(" ");
+        String cmd = parts[0];
+        try {
+            switch (cmd) {
+                case "list":
```

Comment:
Should there be an indentation for the [case clause after the switch statement](https://se-education.org/guides/conventions/java/intermediate.html#:~:text=The%20switch%20statement%20should%20have%20the%20following%20form%3A%20Note%20there%20is%20no%20indentation%20for%20case%20clauses.)?